docs/programming/kotlin/fundamentals/functions.md

```diff
@@ -0,0 +1,11 @@
+# Functions
+
```

Comment:
Would talk about defining new functions too and a little bit background.
See: https://www.composingprograms.com/pages/13-defining-new-functions.html